### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by ghost1face**
Added a .gitignore and more property initializer syntax for capturing the proper 'this' context.  

I am also having a problem running `npm run build` it seems like the browserify command with NODE_ENV=production creates an empty bundle.js file.  Looking to see if I can figure it out.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Logs
```
Comment (tmcw) [inline .gitignore]: Mind keeping this out of the commit? I don't include systematic ignores in any of my projects - I prefer to keep things that are not explicitly unique to the project in my global `.gitignore`

**Comment by ghost1face** [comment]
Removed, I ended up with some logs when trying to figure out the `npm run build` command. But it's not common, removed.

**Comment by tmcw** [comment]
Can you remove it entirely? `node_modules` ignore can/should be in a global gitignore instead of per-repo.

**Comment by ghost1face** [comment]
Done.

**Pull request merged**